js/lib/confirmation-modal.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @package Polylang
+ */
+
+// We can't use underscore or lodash in this common code because it depends of the context classic or block editor.
+// Classic editor underscore is loaded, Block editor lodash is loaded.
+const { __ } = wp.i18n;
+
+const languagesList = jQuery( '#post_lang_choice');
+
+export const bypassConfirmation = () => {
+	let title = jQuery( 'input#title' ).val();
+	let content = jQuery( 'textarea#content' ).val();
+	let excerpt = jQuery( 'textarea#excerpt' ).val();
+	// Block editor case
+	if ( document.getElementById( 'editor' ) && 'undefined' !== typeof wp.data ) {
+		const editor = wp.data.select( 'core/editor' );
+		title = editor.getEditedPostAttribute( 'title' ).trim();
+		content = editor.getEditedPostAttribute( 'content' ).trim();
+		excerpt = editor.getEditedPostAttribute( 'excerpt' ).trim();
+	}
+	return ! title && ! content && ! excerpt;
```

Comment:
Since this function behaves differently whether it is called from the block or classic editor, I don't see benefits from  sharing it.

Reply:
I prefer to have at the same place what is an empty post in the 2 cases.
That allows to have a similar test in classic and block-editor.js script which are enqueued.
It seems to me to be clearer.